examples/persistent-storage/p6/main.cpp

```diff
@@ -19,37 +19,53 @@
 #include "AppConfig.h"
 #include "KeyValueStorageTest.h"
 #include "init_p6Platform.h"
+#include <FreeRTOS.h>
 #include <platform/CHIPDeviceLayer.h>
 #include <platform/KeyValueStoreManager.h>
+#include <task.h>
 
 using namespace chip;
 
-int main(int argc, char * argv[])
+static TaskHandle_t sTestTaskHandle;
+void TestTask(void * pvParameter)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
-    init_p6Platform();
+    err            = chip::DeviceLayer::PersistedStorage::KeyValueStoreMgrImpl().Init();
+    if (err != CHIP_NO_ERROR)
+    {
+        P6_LOG("PlatformMgr().StartEventLoopTask() failed");
```

Comment:
I don't understand this log message here.

Reply:
@bzbarsky-apple : Thanks for review. Updated the comments now.